soul-plugin/soul-plugin-sign/src/main/java/org/dromara/soul/plugin/sign/service/DefaultSignService.java

```diff
@@ -73,7 +74,7 @@
             return Pair.of(Boolean.FALSE, Constants.SIGN_PARAMS_ERROR);
         }
         final LocalDateTime start = DateUtils.formatLocalDateTimeFromTimestamp(Long.parseLong(soulContext.getTimestamp()));
-        final LocalDateTime now = LocalDateTime.now();
+        final LocalDateTime now = LocalDateTime.now(ZoneOffset.ofHours(8));
```

Comment:
If that's bug, Can you file a issue to describe it?

Reply:
OK fix this bug  ,  then I will finished this test future